src/Docfx.App/RunServe.cs

```diff
@@ -41,7 +41,7 @@ public static void Exec(string folder, string host, int? port, bool openBrowser,
             builder.Logging.ClearProviders();
             builder.WebHost.UseUrls(url);
 
-            Console.WriteLine($"Serving \"{folder}\" on {url}. Press Ctrl+C to shut down.");
+            Console.WriteLine($"Serving \"{folder}\" on {url} Press Ctrl+C to shut down.");
```

Comment:
I'm not a project member but hitting this issue I thought a bit on that as well and here are the two outcomes I had:

* it is be a shell configuration (bash in my case)
* that said (since it can be hard to be portable on shells), wouldn't it be better to use a new line instead of a dot to keep the two sentences split and not linked and remove the last dot too?

Hope it makes sense

Reply:
Yeah, really good suggestion to me